08-rails/active-record-migrations.md

```diff
@@ -91,6 +91,21 @@ end
 
 Once we have the migration created, we must again run `rails db:migrate` for the migration to take effect.
 
+## Practice
+
+In our sample Rails project perform the following actions:
+
+1.  Create a new model named: `Author`, an author should only have a `name` field (String).  Remember to run your migration with `rails db:migrate`
+2.  Then add a migration to add a `publisher` field to the model.
```

Comment:
I usually try to say "Create a migration which will add a...." rather than "add a migration" to indicate the way that it needs to create the files rather than adding to an existing file.